packages/adapters-library/src/types/IProtocolAdapter.ts

```diff
@@ -20,6 +20,24 @@ import type {
 } from './adapter'
 import type { Erc20Metadata } from './erc20Metadata'
 
+export type Json =
+  | null
+  | boolean
+  | number
+  | string
+  | Json[]
+  | { [prop: string]: Json }
+
+export type JsonMetadata = Record<string, Json>
+
+export type ProtocolToken<
+  // biome-ignore lint/complexity/noBannedTypes: <explanation>
+  AdditionalMetadata extends JsonMetadata = {},
```

Comment:
I think biome is messing up here, we are not using `{}` as a type, but as a default value.